config/xml_schemas/expected_fails_file.xsd

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+
+  <!-- simple elements -->
+  <xs:element name="category" type="xs:string"/>
+  <xs:element name="status" type="xs:string"/>
+  <xs:element name="issue" type="xs:string"/>
+  <xs:element name="comment" type="xs:string"/>
+
+  <!-- complex elements -->
+
+  <xs:element name="expectedFails">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" maxOccurs="unbounded" ref="test"/>
+      </xs:sequence>
+      <xs:attribute name="version" use="required" type="xs:decimal"/>
+    </xs:complexType>
+  </xs:element>
+
+  <xs:element name="test">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" maxOccurs="1" ref="category"/>
```

Comment:
What is the point of this element? Can a test fail in one category but pass in a different category?

Reply:
@gold2718 - @ekluzek requested that I add this in his review of the companion PR in CTSM (where I changed CTSM's expected fails xml file to this new format): https://github.com/ESCOMP/ctsm/pull/602. My comment from that PR was similar to yours:

> Note that different test suites are now delineated by comments, not by xml sections. This is because, in the machine parsing of this file, we don't have an easy way to determine which category is appropriate, and in any case, it's not really important: if a test from one category is supposed to fail, then it will fail regardless of how we're running the test – including if we're running individual tests, not part of any test suite at all.

@ekluzek do you want to try to justify your thoughts here? Or, based on @gold2718 's query, do you feel okay dropping this element?